package.json

```diff
@@ -17,7 +17,12 @@
     "lint:watch": "npm run lint -- --watch",
     "prepublish": "npm run lint && npm run test && npm run build",
     "test": "mocha --compilers js:babel-register src/**/*-test.js",
-    "test:watch": "npm run test -- --watch src/**/*-test.js"
+    "test:watch": "npm run test -- --watch src/**/*-test.js",
+    "docs:clean": "rimraf _book",
+    "docs:prepare": "gitbook install",
+    "docs:build": "npm run docs:prepare && gitbook build -g acdlite/redux-actions",
+    "docs:watch": "npm run docs:prepare && gitbook serve",
+    "docs:publish": "npm run docs:clean && npm run docs:build && cp CNAME _book && cd _book && git init && git commit --allow-empty -m 'update book' && git checkout -b gh-pages && touch .nojekyll && git add . && git commit -am 'update book' && git push git@github.com:acdlite/redux-actions gh-pages --force"
```

Comment:
Well I guess technically just because you are releasing a new version of the library on npm doesn't necessarily mean you are releasing new documentation. If you want it to be part of that process though I don't see why it couldn't be added to `postpublish`. I added that more for convenience in case you wanted it on `gh-pages` like we planned.